Jenkins fails

Do we need to add dependency on realtime_tools?

http://build.ros.org/job/Kbin_uX64__minas_control__ubuntu_xenial_amd64__binary/29/

```
/usr/lib/ccache/x86_64-linux-gnu-g++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"minas_control\" -I/tmp/binarydeb/ros-kinetic-minas-control-1.0.9/include -I/opt/ros/kinetic/include -I/opt/ros/kinetic/share/xmlrpcpp/cmake/../../../include/xmlrpcpp -I/opt/ros/kinetic/include/soem  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -DNDEBUG -Wdate-time -D_FORTIFY_SOURCE=2    -o CMakeFiles/main.dir/src/main.cpp.o -c /tmp/binarydeb/ros-kinetic-minas-control-1.0.9/src/main.cpp
/tmp/binarydeb/ros-kinetic-minas-control-1.0.9/src/main.cpp:60:47: fatal error: realtime_tools/realtime_publisher.h: No such file or directory
compilation terminated.
CMakeFiles/main.dir/build.make:65: recipe for target 'CMakeFiles/main.dir/src/main.cpp.o' failed
make[4]: *** [CMakeFiles/main.dir/src/main.cpp.o] Error 1
```

--- a/minas_control/CMakeLists.txt
+++ b/minas_control/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(minas_control)
 
-find_package(catkin REQUIRED COMPONENTS controller_manager diagnostic_updater transmission_interface hardware_interface sensor_msgs joint_limits_interface trajectory_msgs ethercat_manager soem)
+find_package(catkin REQUIRED COMPONENTS controller_manager diagnostic_updater transmission_interface hardware_interface sensor_msgs joint_limits_interface trajectory_msgs ethercat_manager soem realtime_tools)
 
 ###################################
 ## catkin specific configuration ##

--- a/minas_control/package.xml
+++ b/minas_control/package.xml
@@ -21,6 +21,7 @@
   <build_depend>ethercat_manager</build_depend>
   <build_depend>hardware_interface</build_depend>
   <build_depend>joint_limits_interface</build_depend>
+  <build_depend>realtime_tools</build_depend>
   <build_depend>sensor_msgs</build_depend>
   <build_depend>soem</build_depend>
   <build_depend>trajectory_msgs</build_depend>
@@ -32,6 +33,7 @@
   <run_depend>ethercat_manager</run_depend>
   <run_depend>hardware_interface</run_depend>
   <run_depend>joint_limits_interface</run_depend>
+  <run_depend>realtime_tools</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>trajectory_msgs</run_depend>
   <run_depend>transmission_interface</run_depend>

